lib/yu.rb

```diff
@@ -220,7 +220,7 @@ def get_existing_services(service_names)
       service_names.select { |service_name| Pathname(service_name).exist? }
     end
 
-    def copy_template_into_dir(service_name:)
+    def copy_template_into_dir(service_name)
```

Comment:
Because this is a named param with no default value, I would make this `options` and do `options.fetch(:service_name)`